autofill/autofill-impl/src/test/java/com/duckduckgo/autofill/impl/SecureStoreBackedAutofillStoreTest.kt

```diff
@@ -412,6 +415,67 @@ class SecureStoreBackedAutofillStoreTest {
         }
     }
 
+    @Test
+    fun whenUpdatingPasswordWithUsernameInDifferentCasingThenOriginalUsernameUntouched() = runTest {
```

Comment:
I think this test just validate the opposite to the 4th scenario in the test cases:

- Visit https://autofill.me/form/registration-username
- For username, enter USERname (match the case exactly)
- For password, enter something different from what’s stored (e.g., new-password or autogenerate it)
- Tap Register button. Verify you are prompted to Update password for the username
- Agree. Now visit password management page and verify the password has been updated for all 3 entries (by design, it updates multiple matches)

Seems we expect to update all entries, and in the unit test is expected to keep the original.

Reply:
>I think this test just validate the opposite to the 4th scenario in the test cases

No, this is different. 
- the scenario from the PR description is about checking the **password** is updated.
- this unit test scenario is about ensuring the **username** is not updated (only the password is updated)

Reply:
Discussed offline and password is updated. The username is not, which is the intended behavior.